### DISCUSSION TITLE
pathofthebikehorn1 option doesnt have endconversation="true"

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14273] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Can't close the text box after refusing.

Factionevents.xml
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/e1ad2481-44f0-4d1f-96e8-75638ca72886)

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/3b50eaab-85f1-4443-8fdc-6a0644fe8b50)

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
Thanks for the report, a fix is in the works. In the future we'll likely opt for a code solution so these dead ends don't happen as easily anymore.

**Comment by Regalis11** [comment]
This has been addressed since the report was created, looks like we just forgot to close the ticket. :)
